src/main.js

```diff
@@ -69,16 +86,31 @@ const headerElement = document.querySelector('.header');
 const mainElement = document.querySelector('.main');
 renderElement(headerElement, new UserStatusView().getElement(), RenderPosition.BEFOREEND);
 renderElement(mainElement, new MenuView(dataFilms).getElement(), RenderPosition.BEFOREEND);
-renderElement(mainElement, new SortView().getElement(), RenderPosition.BEFOREEND);
+dataFilms.length ? renderElement(mainElement, new SortView().getElement(), RenderPosition.BEFOREEND) : null;
```

Comment:
тут не нужен тернарник, тут нужен `if` вот короткая версия
```suggestion
dataFilms.length && renderElement(mainElement, new SortView().getElement(), RenderPosition.BEFOREEND);
```